WordPress/Classes/ViewRelated/NUX/LoginViewController.swift

```diff
@@ -71,9 +71,17 @@ class LoginViewController: NUXViewController, SigninWPComSyncHandler, LoginFacad
 
     func dismiss() {
         if shouldShowEpilogue() {
-            self.performSegue(withIdentifier: .showEpilogue, sender: self)
-            return
+            if let linkSource = loginFields.meta.emailMagicLinkSource {
```

Comment:
If there is no `emailMagicLinkSource` then the epilogue doesn't appear at all. For example, with self-hosted logins.